mlflow/utils/cli_args.py

```diff
@@ -32,6 +32,12 @@
                                    " activated. The version of installed mlflow will be the same as"
                                    "the one used to invoke this command.")
 
+BASE_IMAGE = click.option("--base-image", "-b", default=None,
+                          help="Name of the base image.")
+
+NO_JAVA = click.option("--no_java", is_flag=True,
+                       help="Do not include Java libraries or start the Java REST server.")
```

Comment:
reword help as

```
Do not install Java libraries and packages and do not start Java based backend server(s).
```

Perhaps Java "libraries" and "servers" could be different options in future.

cc: @smurching @dbczumar @stbof

Reply:
Do not install Java libraries and do not start Java-based servers.